portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact(){
+    const facts =
+    [
+    'I am currently learning russian,Привет ', 
+    'I am interested in compilers and progamming languages',
+    'I have lived in three states: California, New York, and Virginia !'
+    ];
+
+    // Get a random fact.
+      const fact = facts[Math.floor(Math.random() * facts.length)];
```

Comment:
Design question about this. Is it okay for the user to see the same fact multiple times after pressing the button or do you always want it to show something different?

Reply:
I ultimately would like to give unique facts. One solution I thought was to potentially store the facts about me in a set in comparison to an array,but, I am unsure as to configure my html so that when the button is pressed it pops from the set ?

Reply:
A few possibilities:

- You could shuffle the array and then return each element one-by-one
- You could keep track of the last generated random number and regenerate if you get the same thing. (non-zero chance of stalling forever though) 
- You could keep track of the last generated random number and always return the next element if you regenerate the same number, or return the first element if the previously generated number is the last index in the array.

Reply:
I'm currently leaning towards either option 2 or 3. Ack !